src/services/admin/admin.tsx

```
@@ -0,0 +1,101 @@
+					Authorization: `Bearer ${localStorage.getItem('authToken')}`,
+				},
+			}
+		);
+		return response.data;
+	} catch (error) {
+		console.error('Error fetching document field mapping:', error);
+		throw error;
+	}
+};
+
+export const fetchFields = async (context = 'USERS', contextType = 'User'): Promise<Field[]> => {
```

ESLint

[error] Replace `context·=·'USERS',·contextType·=·'User'` with `⏎↹context·=·'USERS',⏎↹contextType·=·'User'⏎`

(prettier/prettier)